README.md

```diff
@@ -45,58 +45,67 @@ Read [FAQs](https://github.com/acidanthera/WhateverGreen/blob/master/Manual/) an
 
 | Boot argument 	| DeviceProperties 	| Description 	|
 |---	|---	|---	|
-| `-wegdbg` 	| N/A 	| Enable debug printing (available in DEBUG binaries) 	|
-| `-wegoff` 	| N/A 	| Disable WhateverGreen 	|
-| `-wegbeta` 	| N/A 	| Enable WhateverGreen on unsupported OS versions (13 and below are enabled by default) 	|
-| `-wegnoegpu` 	| `disable-gpu` property to each GFX0 	| Disable all external GPUs 	|
-| `-wegnoigpu` 	| `disable-gpu` property to IGPU 	| Disable internal GPU 	|
-| `-wegswitchgpu` 	| `switch-to-external-gpu` property to IGPU 	| Disable internal GPU when external GPU is installed 	|
-| `-radvesa` 	| N/A 	| Disable ATI/AMD video acceleration completely 	|
-| `-rad24` 	| N/A 	| Enforce 24-bit display mode 	|
-| `-raddvi` 	| N/A 	| Enable DVI transmitter correction (required for 290X, 370, etc.) 	|
-| `-radcodec` 	| N/A 	| Force the spoofed PID to be used in AMDRadeonVADriver 	|
-| `radpg=15` 	| N/A 	| Disable several power-gating modes (see [FAQ Radeon](https://github.com/dreamwhite/WhateverGreen/blob/master/Manual/FAQ.Radeon.en.md), required for Cape Verde GPUs) 	|
-| `agdpmod=vit9696` 	| `agdpmod` property to external GPU 	| Disable check for `board-id` 	|
-| `agdpmod=pikera` 	| `agdpmod` property to external GPU 	| Replaces `board-id` with `board-ix` 	|
+| Generic			    |
```

Comment:
I'd say to split the sections onto different tables using markdown `#`syntax

Reply:
@dreamwhite 
Done on my side, I have fixed board-id and 2nd stage boot headers.